storage-access.bs

```diff
@@ -144,6 +178,8 @@ Modify the definition of [=source snapshot params=] in the following manner:
 1. Add a new member called <dfn for="source snapshot params">has storage access</dfn> of type [=boolean=].
 1. Add a new member called <dfn for="source snapshot params">environment id</dfn> of type opaque [=string=].
 
+Add a new global <dfn for="storage access eligible connections set">storage access eligible connections set</dfn> of type [=ordered set=] on the [=user agent=]. Its [=list/items=] are [=tuples=] of the form (|embedder site|, |identity provider site|) where |embedder site| and |identity provider site| are [=site|sites=]. It represents the set of (|embedder site|, |identity provider site|) tuples for which |identity provider site| invoked {{Document/requestStorageAccess}} while embedded under |embedder site|, where the browser allowed the invocation to resolve due to |embedder site| and |identity provider site| being present in the [=connected accounts set=].
```

Comment:
As discussed, I think this describes more of an implementation detail (the set is used in Chromium to avoid trusting the renderer process about the hasStorageAccess bit) which can be abstracted away in the spec. However, it seems like a reasonable idea to turn it into a brief non-normative note for implementers.

Reply:
Makes sense, done.